ruby/ql/lib/codeql/ruby/security/LdapInjectionQuery.qll

```diff
@@ -0,0 +1,26 @@
+/**
+ * Provides default sources, sinks and sanitizers for detecting
+ * LDAP Injections, as well as extension points for adding your own
+ */
+
+private import codeql.ruby.DataFlow
+private import codeql.ruby.TaintTracking
+private import LdapInjectionCustomizations
+private import LdapInjectionCustomizations::LdapInjection
+
+/**
+ * A taint-tracking configuration for detecting LDAP Injections vulnerabilities.
+ */
+class Configuration extends TaintTracking::Configuration {
```

Comment:
Would you mind converting this to a `DataFlow::ConfigSig` as in https://github.com/github/codeql/pull/13130/files#diff-cb35acf3aed06823bd9b005af3d73697a4cb92867d123eb7b868f9b6143c4082 (using `isBarrier` and `isAdditionalFlowStep` in place of `isSanitizer` and `isAdditionalTaintStep` respectively).